tests/node-test.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+
+kiele vm --port 9001 &
+sleep 3
+make test-iele-node  "$@" TEST_PORT=9001
+make test-bad-packet "$@" TEST_PORT=9001
+kill %kiele
```

Comment:
Allow passing in `TEST_PORT` as an argument, and have it be a `Makefile` variable as well, so that users can call `make test-node TEST_PORT=9003` if they need to.